dd-sdk-android/src/test/kotlin/com/datadog/android/core/internal/time/KronosTimeProviderTest.kt

```diff
@@ -67,6 +67,18 @@ internal class KronosTimeProviderTest {
         )
     }
 
+    @Test
+    fun `returns server time offset in milliseconds`() {
+        val now = System.currentTimeMillis()
+        val result = testedTimeProvider.getServerOffsetMillis()
+
+        val expectedOffset = fakeDate.time - now
+        assertThat(result).isCloseTo(
+            expectedOffset,
+            Offset.offset(50L)
```

Comment:
why here we allow `50 ms` offset, but in the test above it is just `1 ms`?

Reply:
Yes...I remember this test, It didn't work with 1 millisecond for some reason it was too flaky when I ran it multiple times in a row and I wanted to make sure. I think I can lower that offset though 👍

Reply:
Actually I remembered why, there is a test bellow from previous commits : `returns device time` which uses the same offset and I just followed that. It is true that I first tried with `1ms` offset it did not work but honestly it is weird indeed but then I've seen the previous test where 50ms where used as offset and just conformed to that. I will try to test more to see what's going on.